exchanges/assets.py

```diff
@@ -0,0 +1,49 @@
+class Asset(object):
+
+    def __init__(self, name, atype, iso4217_a3=None, abbreviation=None):
+
+        if iso4217_a3 is None and abbreviation is None:
+            raise ValueError('At least one denomination is needed')
```

Comment:
We will probably never see this error, but the message is not clear to me. Maybe add 'You must provided either an abbreviation or the iso4217_a3 name.'